modules/customizing-the-appearance/proc-customize-rhdh-sidebar-menuitems.adoc

```diff
@@ -1,129 +1,60 @@
 [id='proc-customize-rhdh-sidebar-menuitems_{context}']
 = Customizing the sidebar menu items for your {product-short} instance
 
-The sidebar menu in {product} consists of two main parts:
+The sidebar menu in {product} consists of two main parts that you can configure:
 
-* *Main menu items*: These items are the static menu items that form the core navigation structure of sidebar. These menu items remain consistent and are predefined.
-
-* *Dynamic plugin menu items*: These items are displayed beneath the main menu and can be customized based on the plugins installed. The main menu items section is dynamic and can change based on your preferences and installed plugins.
+* *Dynamic plugin menu items*: Your preferences and your active plugins define dynamically one part of the sidebar menu.
+* *Main menu items*: The core navigation structure of sidebar is static.
 
 .Procedure
 
-. Customize the main menu items using the following steps:
+. To configure *Dynamic plugin menu item*, add the following content to your `{my-app-config-file}` file:
 +
---
-.. Open the `app-config-rhdh.yaml` file.
-.. To customize the order and parent-child relationships for the main menu items, use the `dynamicPlugins.frontend.default.main-menu-items.menuItems` field.
-.. For dynamic plugin menu items, use the `dynamicPlugins.frontend.<package_name>.menuItems` field.
-
-.Example `app-config-rhdh.yaml` file
 [source,yaml]
 ----
 dynamicPlugins:
   frontend:
-    <package_name>: # same as `scalprum.name` key in plugin's `package.json`
-      menuItems: # optional, allows you to configure plugin menu items in the main sidebar navigation
-        <menu_item_name>: # unique name in the plugin menu items list <1>
-          icon: home | group | category | extension | school | _<your_icon>_  # <2>
-          title: My Plugin Page # optional, same as `menuItem.text` in `dynamicRoutes` <3>
-          priority: 10 # optional, defines the order of menu items in the sidebar <4>
-          parent: favorites # optional, defines parent-child relationships for nested menu items <5>
-----
-
-You can modify the fields in the previous example to configure the desired order and parent-child relationships of the sidebar menu items.
-
-<1> This attribute represents a unique name in the main sidebar navigation. It can denote either a standalone menu item or a parent menu item. If this attribute represents a plugin menu item, the name of the attribute must match with the corresponding path in `dynamicRoutes`. For example, if `dynamicRoutes` defines `path: /my-plugin`, then `menu_item_name` must be defined as `my-plugin`.
+    <package_name>: # <1>
+      menuItems: # <2>
```

Comment:
Omit teh admonition. There is nothing to edit here for the user, therefore it is more logical to move the description in the step general text.

```suggestion
      menuItems:
```

(Of course then you need to renumber the admonitions.)